### ISSUE TITLE
ESPnet2 transducer: four losses

### ISSUE THREAD

**Issue by wyunj** [espnet/espnet#4562]
Hello, I wonder if ESPnet2 supports multi-task learning with various auxiliary losses (four losses), such as: CTC, cross-entropy w/ label-smoothing (LM loss), auxiliary Transducer, and symmetric KL divergence?
Thanks in advance!

**Comment by b-flo** [comment]
Hi,

Currently, you can only train ESPnet2 Transducer in pair with CTC. Total loss being defined by:
`tot_loss = loss_trans + λ_CTC * loss_CTC`).

Another version will be added in the future with support for auxiliary LM loss and weighting Transducer loss. The other auxiliary tasks are considered but we're currently trying some things related to that, it may take another form in the end.
